src/Entity/Notification.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\NotificationRepository;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: NotificationRepository::class)]
+class Notification
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private ?int $id = null;
+
+    #[ORM\ManyToOne(inversedBy: 'notifications')]
+    #[ORM\JoinColumn(nullable: false)]
+    private ?NotificationDefinition $definitionId = null;
+
+    #[ORM\Column]
+    private ?bool $status = null;
```

Comment:
Z tego co widzę to nigdzie nie wykorzystujesz statusu. Poza tym status powinien być enumem. Poczytaj jak wykorzystać enum w doctrine entity. Możliwe statusy to powinny być myślę:

CREATED - po stworzeniu notyfikacji ale przed wysłaniem
SUCCESS - po wysłaniu notyfikacji
FAILED - gdy wystąpi jakiś błąd przy wysyłce notyfikacji